Sources/MockoloFramework/Utils/StringExtensions.swift

```diff
@@ -70,6 +78,27 @@ extension String {
     var isThrowsOrRethrows: Bool {
         return self == .throws || self == .rethrows
     }
+
+    var safeName: String {
+        let kewords: [String] = [
```

Comment:
declare this as a static let keywords array -- otherwise it'll create the list every time